README.md

```diff
@@ -73,9 +73,12 @@ found in the Node.js
 [documentation](https://nodejs.org/docs/latest-v14.x/api/buffer.html).
 
 ```js
-module.exports.fuzz = function (data) {
-	const intParam = data.readInt32BE(0);
-	const stringParam = data.toString("utf-8", 4);
+const { FuzzedDataProvider } = require("@jazzer.js/core");
```

Comment:
Could you also adjust the paragraph above and describe how `FuzzedDataProvider` could be used as a more convenient alternative to the raw `Buffer`?

Perhaps a section in the `Advanced Fuzzing Settings` docs would also be nice.